Dockerfile

```diff
@@ -24,4 +30,5 @@ RUN /usr/bin/kubectl version -c
 RUN /usr/bin/fleetctl version
 RUN /usr/bin/coreos-cloudinit -version
 RUN /usr/bin/s3secrets --help > /dev/null
+RUN /usr/bin/kb8or --version > /dev/null
```

Comment:
Maybe let the version output be visible, so we know what got installed if there is a problem.